src/utils/node.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { Port } from '../types';
+import { isPortAvailable } from './network';
+import { execAsync } from './process';
+
+export const installLiskCore = async (): Promise<string> => execAsync('npm i -g lisk-core');
```

Comment:
May be we can define command as constant and use it here:
```suggestion
const INSTALL_LISK_CORE_COMMAND = 'npm i -g lisk-core';
export const installLiskCore = async (): Promise<string> => execAsync(INSTALL_LISK_CORE_COMMAND);
```

Reply:
Applied.